App/components/Input/input.story.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import styled from 'styled-components/native'
+import { action } from '@storybook/addon-actions'
+import { boolean, select, text, number } from '@storybook/addon-knobs'
+import { storiesOf } from '@storybook/react-native'
+import { BufferView, ThemeDecorator } from '../../config/decorators'
+import { Input } from './input'
+import { INPUT_SIZE_KEYS } from '../../theme/inputSizes'
+
+const StyledWrapper = styled.View`
+  display: flex;
+  flex-direction: column;
+`
+
+const StyledPadding = styled.View`
+  margin-top: 5px;
+  margin-bottom: 5px;
+`
+
+export const list = () => (
+  <StyledWrapper>
+    <Input
+      onChange={() => 's'}
+      size='lg'
+      clearButtonMode
+      placeholder='Поиск по избранному'
+      editable={false}
+    />
+    <StyledPadding />
+    <Input
+      onChange={() => 's'}
+      size='md'
+      clearButtonMode
+      placeholder='Поиск по избранному'
+      editable
+    />
+    <StyledPadding />
+    <Input
+      onChange={() => 's'}
+      size='sm'
+      clearButtonMode
+      placeholder='Поиск по избранному'
+      editable
+    />
+    <StyledPadding />
+    <StyledPadding />
+    <Input
+      onChange={() => 's'}
+      size='lg'
+      clearButtonMode={false}
+      placeholder='Поиск по избранному'
+      editable={false}
+    />
+    <StyledPadding />
+    <Input
+      onChange={() => 's'}
+      size='md'
+      clearButtonMode={false}
+      placeholder='Поиск по избранному'
+      editable
+    />
+    <StyledPadding />
+    <Input
+      onChange={() => 's'}
+      size='sm'
+      clearButtonMode={false}
+      placeholder='Поиск по избранному'
+      editable
+    />
+  </StyledWrapper>
+)
+
+export const knobs = () => (
+  <StyledWrapper>
```

Comment:
Please add setState() so we can use onChange func

Reply:
done